core/src/main/java/utilcalc/core/pdfGen/HtmlBuilder.java

```diff
@@ -0,0 +1,69 @@
+package utilcalc.core.pdfGen;
+
+public final class HtmlBuilder {
+
+    private final StringBuilder sb = new StringBuilder();
+
+    public HtmlBuilder() {
+        sb.append(
+                """
+            <!DOCTYPE html>
+            <html xmlns="http://www.w3.org/1999/xhtml">
+            <head>
+                <meta charset="UTF-8" />
+                <style type="text/css">
+                    body { font-family: 'DejaVu Sans', sans-serif; margin: 40px; }
+                    h1 { font-size: 20pt; }
+                    p { font-size: 12pt; margin: 4pt 0; }
+                    .italic { font-style: italic; font-size: 10pt; }
+                </style>
+            </head>
+            <body>
+        """);
+    }
+
+    public HtmlBuilder h1(String text) {
+        sb.append("<h1>").append(escape(text)).append("</h1>\n");
+        return this;
+    }
+
+    public HtmlBuilder p(String text) {
+        sb.append("<p>").append(escape(text)).append("</p>\n");
+        return this;
+    }
+
+    public HtmlBuilder pItalic(String text) {
+        sb.append("<p class=\"italic\">").append(escape(text)).append("</p>\n");
+        return this;
+    }
+
+    public HtmlBuilder pItalicIndented(String text, int indentPx) {
```

Comment:
Proc se na italic pouziva class atribut, ale na margin se ppouziva style? Muzeme to sjednotit?